InnerEye/ML/config.py

```diff
@@ -480,6 +480,12 @@ class SegmentationModelBase(ModelConfigBase):
                                                  "of the first N subjects in the training set will be "
                                                  "written to the outputs folder.")
 
+    #: Some loss functions (e.g. SoftDice) may produce results on overlapping labels, but others (e.g. FocalLoss) will
+    #: fail with a cryptic error message. If you are sure that you want to use mutually exclusive labels then set this
+    #: to False and the check that the labels are mutually exclusive in
+    #: InnerEye.ML.utils.io_util.load_labels_from_dataset_source will not run.
+    check_exclusive: bool = param.Boolean(True, doc="Raise an error if the segmentation labels are not mutually exclusive.")
```

Comment:
I generally advise to describe the behaviour in both True and False case. "If True, raise an error if... If False, do not check labels. In that case, the loss functions may raise an exception". Or something alike.

Reply:
Good call. Re-worded.